File: Shared/Requests/Common/AddressRequest.cs
﻿namespace Shared.Requests.Common;

public class AddressRequest
{
    public Guid ParentId { get; set; }
    public Guid Id { get; set; }
    public string ContactNumber { get; set; } = string.Empty;
    public string AddressLine1 { get; set; } = string.Empty;
    public string AddressLine2 { get; set; } = string.Empty;
    public string City { get; set; } = string.Empty;
    public string District { get; set; } = string.Empty;
    public string Pincode { get; set; } = string.Empty;
    public string State { get; set; } = string.Empty;
    public string Country { get; set; } = string.Empty;
}

File: WebUI/src/app/_requests/address-request.ts
import { Guid } from "guid-typescript";

export class AddressRequest {
    id: Guid;
    parentId: Guid;
    attention: string;
    contactNumber: string;
    addressLine1: string;
    addressLine2: string;
    city: string;
    pinCode: string;
    state: string;
    country: string;
}
